tests/DocumentationExamplesTest.php

```diff
@@ -1241,12 +1266,12 @@ public function testTransactions_intro_example_1()
         $client = new Client(static::getUri());
 
         /* The WC is required: https://docs.mongodb.com/manual/core/transactions/#transactions-and-locks */
-        $client->hr->dropCollection('employees', ['writeConcern' => new \MongoDB\Driver\WriteConcern('majority')]);
-        $client->reporting->dropCollection('events', ['writeConcern' => new \MongoDB\Driver\WriteConcern('majority')]);
+        $client->hr->dropCollection('employees', ['writeConcern' => new WriteConcern('majority')]);
+        $client->reporting->dropCollection('events', ['writeConcern' => new WriteConcern('majority')]);
 
         /* Collections need to be created before a transaction starts */
-        $client->hr->createCollection('employees', ['writeConcern' => new \MongoDB\Driver\WriteConcern('majority')]);
-        $client->reporting->createCollection('events', ['writeConcern' => new \MongoDB\Driver\WriteConcern('majority')]);
+        $client->hr->createCollection('employees', ['writeConcern' => new WriteConcern('majority')]);
+        $client->reporting->createCollection('events', ['writeConcern' => new WriteConcern('majority')]);
```

Comment:
I think these should be left as-is.

Reply:
I've excluded namespace changes for all code within the doc examples, but changed everything outside of docs examples to match our coding standard. This code is outside of the examples so I've updated it to match the rest of the code base.

Reply:
My mistake. I originally thought this was inside rendered example code.